tests/error/linear_errors/unused_field1.err

```diff
@@ -0,0 +1,6 @@
+Guppy compilation failed. Error in file $FILE:18
+
+16:    @guppy(module)
+17:    def foo(s: MyStruct) -> int:
+               ^^^^^^^^^^^
+GuppyError: Field `s.q` with linear type `qubit` is not used on all control-flow paths
```

Comment:
The above errors where found while checking a single BB. This one is found via global dataflow analysis since there is an unconditional branch to the return BB.

We currently always add the `"on all control-flow paths"` suffix if errrors are found using the latter method, even if the branches are unconditional. I think this wouldn't be too hard to fix, but again adding it to #306 since it's not directly related to structs